selfdrive/ui/ui.cc

```diff
@@ -226,6 +259,12 @@ UIState::UIState(QObject *parent) : QObject(parent) {
   timer = new QTimer(this);
   QObject::connect(timer, &QTimer::timeout, this, &UIState::update);
   timer->start(1000 / UI_FREQ);
+
+  // initial dm draw
```

Comment:
what's this for? don't we hide the icon until the DM packet is seen?

Reply:
no? we only hide dm on alerts

Reply:
made a dm_rendered flag instead. This way the icon never shows if dm model packet for whatever reason doesn't show up